warehouse/rss/views.py

```diff
@@ -46,6 +48,43 @@ def rss_updates(request):
 
 
 @view_config(
+    route_name="rss.project_updates",
+    renderer="rss/project_updates.xml",
+    decorator=[
+        origin_cache(
+            1 * 24 * 60 * 60,                         # 1 day
+            stale_while_revalidate=1 * 24 * 60 * 60,  # 1 day
+            stale_if_error=5 * 24 * 60 * 60,          # 5 days
+        ),
+    ],
+)
+def rss_project_updates(request):
+    project_name = request.matchdict["name"]
+    request.response.content_type = "text/xml"
+
+    request.find_service(name="csp").merge(XML_CSP)
+
+    latest_releases = (
+        request.db.query(Release)
+        .join(Project)
+        .filter(
+            Project.normalized_name == func.normalize_pep426_name(project_name)
+        )
+        .order_by(Release.created.desc())
+        .limit(10)
+        .all()
+    )
+
+    if not latest_releases:
+        return HTTPNotFound()
+
+    return {
+        "project_name": project_name,
+        "latest_releases": latest_releases
+    }
```

Comment:
If we made the necessary changes to get a `project` via the `ProjectFactory`, we could just pass the `project` variable to the template here, and use `project.name` and `project.releases` (which is ordered by creation date) directly in the template.